host/2.0/stretch/amd64/dotnet/dotnet-slim.Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
     mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
     rm -rf /root/.local /root/.nuget /src
 
-RUN EXTENSION_BUNDLE_VERSION=1.5.0 && \
+RUN EXTENSION_BUNDLE_VERSION=1.6.0 && \
     EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.5.0_linux-x64.zip && \
```

Comment:
It looks wrong. Make it 1.6.0 

This is the error message of CI

```
https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.6.0/Microsoft.Azure.Functions.ExtensionBundle.1.5.0_linux-x64.zip 
```

Reply:
Thank you @TsuyoshiUshio , this was indeed the fix :)